llvm/test/Transforms/VectorCombine/RISCV/vector-interleave2-splat.ll

```diff
@@ -0,0 +1,14 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S -mtriple=riscv64 -mattr=+v,+m,+zvfh %s -passes=vector-combine | FileCheck %s
+; RUN: opt -S -mtriple=riscv32 -mattr=+v,+m,+zvfh %s -passes=vector-combine | FileCheck %s
```

Comment:
Add a test that uses Zve32x instead of V. We shouldn't form an i64 vector type in that case.

Reply:
Maybe test to make sure we don't do i64->i128 too.

Reply:
Both are done.